docs/blog/newsletter-november-2021.rst

```diff
@@ -0,0 +1,137 @@
+
+------------------------------------------------
+Managing when your manager lacks docs experience
+------------------------------------------------
+
+This month, our documentarians discussed how they have handled differences with a manager who has no experience with documentation. Among those who have worked for managers without a docs background, there were roughly two groups: the manager either deferred to the technical writer's expertise or did not.
+
+If your manager is in the first category, you'll likely encounter less heavyhanded supervision, and docs-related disagreements may be rare. If your manager in the second category, you have a harder path to tread, especially when your manager's perspective on docs-related tasks doesn't quite match yours.
+
+In the course of this conversation, we learned a new acronym: HiPPO, meaning "highest paid person's opinion." In this context, the HiPPO method means ultimately disagreements between tech writers and managers are resolved according to the more senior person's opinion, regardless of whether they have docs experience. When you are subject to the HiPPO method, your ability to affect docs-related decisions depends on how firmly you can gain your manager's trust.
+
+Even when your manager is inclined to trust your ability, you'll still need to confirm their confidence by doing good work, cooperating with the rest of the team, and meeting deadlines. This approach is the best path you can take with managers who don't grasp your ability either -- demonstrating that you understand and can document complex processes might be the only way you can get traction.
+
+---------------------------
+Options for single-sourcing
+---------------------------
+
+Single-source publishing is a content management technique that lets you use one source of content but publish it to different output formats (“write once, publish everywhere”). It's not something that every team needs, but it can be super useful if you need to publish to the web *and* PDFs, or if you have many similar variants on one product. If your team is not “technical” - more specifically if they are unfamiliar with version control software like Git, lack software development skills, or have no way of obtaining the support necessary to implement a docs-as-code approach, component content management systems (CCMS) like Paligo and MadCap Flare might be more suitable.
+
+Regardless of the method you choose, decide on a single-source publishing strategy and consider the advantages and disadvantages of each option to determine which is ideal for your team. Distributed contributions, for example, are maybe a reason to choose the docs-as-code method. In contrast, if you use Flare or a docs-focused CMS like Knowledge Owl, you'll find more robust support for reusing articles or snippets. Whatever method or tool you choose, have a plan in place to ensure a successful single-source publishing approach.
```

Comment:
Should it be "...for example, may be a reason..." rather than "...for example, are maybe a reason..." perhaps?

Reply:
Much better, added!